libavpipe/src/avpipe_xc.c

```diff
@@ -390,6 +390,23 @@ decode_interrupt_cb(
     return decoder_ctx->cancelled;
 }
 
+static int
+check_stream_index(
+    xcparams_t *params,
+    coderctx_t *decoder_context)
+{
+    for (int i = 0; i < decoder_context->format_context->nb_streams && i < MAX_STREAMS; i++) {
```

Comment:
This for loop is unnecessary, as you don't actually use `i`.

Reply:
ooops!